janitor/math.py

```diff
@@ -0,0 +1,205 @@
+""" Miscellaneous mathematical operators. """
+
+import datetime as dt
+import functools
+import os
+import re
+import sys
+import unicodedata
+import warnings
+from fnmatch import translate
+from functools import partial, reduce
+from typing import Any, Callable, Dict, Iterable, List, Tuple, Union
+
+import numpy as np
+import pandas as pd
+import pandas_flavor as pf
+from scipy.special import expit
+from scipy.stats import mode, norm
+from sklearn.preprocessing import LabelEncoder
+
+from .errors import JanitorError
+from .utils import (
+    _clean_accounting_column,
+    _currency_column_to_numeric,
+    _replace_empty_string_with_none,
+    _replace_original_empty_string_with_none,
+    _strip_underscores,
+    check,
+    check_column,
+    deprecated_alias,
+)
+
+
+@pf.register_series_method
+def log(s: pd.Series, error: str = "warn") -> pd.Series:
+    """
+    Take natural logarithm of the Series
+
+    Parameters
```

Comment:
For docstring style, I think we have been sticking to sphinx style, rather than NumPy style. Would you be kind enough to reformat the docstrings please?

I will raise copy/pasted comments at the relevant sections below so that we can use it as a task list tracker-ish thing.